ci-operator/step-registry/ipi/conf/azurestack/additional-ca/ipi-conf-azurestack-additional-ca-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ ! -f "${CLUSTER_PROFILE_DIR}/ash-wwt-ca.pem" ]]; then
```

Comment:
When calling the step by users, the step is to inject additional ca, so I guess it should be exit as failure, we can remove the condition pre-check now.